test/basic.html

```diff
@@ -51,6 +51,10 @@
             assert.equal(pages.selected, undefined);
           });
 
+          test('null activateEvent', function() {
```

Comment:
This test asserts that `activeEvent` is null, but does not describe what was happening when `activeEvent` was something other than `null`. If I were looking at this test for the first time, it would require a lot of extra exploration - probably leading me all the way back to `IronSelectableBehavior` - to figure out what is actually being tested here.